### ISSUE TITLE
instance stop doesn't work

### ISSUE THREAD

**Issue by IschdeFelin** [modrinth/code#218]
### Describe the bug

If you have an instance running and press stop, nothing happens.

### Steps to reproduce

1. Open an instance
2. Run the instance
3. Press the stop button in the instance view

### Expected behavior

I expect that the instance will stop.

**Comment by thesuzerain** [comment]
Having trouble reproducing this one. Can you give me some more information?

1. What OS are you on? Mac, Linux, Windows, WSL, etc, what version?
2. Does this happen with any instance? Even just an empty vanilla one? 
3. are you stopping it immediately before it opens? Or letting the game open
4. is the button responsive when you press it? Does the launcher seem to think it closed (ie: 'stop' changes back to 'play',the instance playing disappears from the top right bar of the launcher and returns to 'No intance running', etc) without the instance itself stopping?
5. Are you only running one instance at a time, or multiple ?

**Comment by IschdeFelin** [comment]
1) I'm using Windows 11 Home 22H2.
2) It happens with every instance. I tried Vanilla, Fabric, Forge and Quilt and also with Mods installed.
3) I have tested both scenarios, where I stop it immediately after opening and also where I let the game open for a while before stopping it. The issue occurs in both cases.
4) Yes, the button switches back to 'Play' and on the top right is also displayed 'No running instances' but the instance doesn't stop.
5) I have tried running both single and multiple instances at a time, but the issue persists regardless.
